### PR TITLE
Shimahaj

### PR DESCRIPTION
**Pull request by codevoyage**
Non-autoblocked version of the banner. Draft PR, since autoblocking is WIP. 

First PR on first EDS project, new to frontend, creating draft PR to find out if on the right path and approach.

Authoring experience: https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B7E242C38-9AC2-4EED-A99C-9F0938595168%7D&file=Document.docx&action=default&mobileredirect=true

Fix https://github.com/aemsites/hubblehomes-com/issues/9

Test URLs:
- Before: https://www.hubblehomes.com/
- After: https://shimahaj--hubblehomes-com--aemsites.hlx.page/drafts/shimahaj/document

### CODE REVIEW

**Review by varunmitra — CHANGES_REQUESTED** [review]
@codevoyage please clean up this PR. There are 35 files listed here. Please merge the latest changes from the main branch

**Review by varunmitra — CHANGES_REQUESTED** [review]
Screenshot 2024-05-21 at 10 19 13 AM
There is a slight mis match for the form `:focus` element state. Original form element has the following styling:
 `.form-control:focus {
    border-color: #66afe9;
    outline: 0;
    box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075), 0 0 8px rgba(102, 175, 233, .6);`

**Comment by codevoyage** [comment]
> @codevoyage please clean up this PR. There are 35 files listed here. Please merge the latest changes from the main branch

Yes, sorry about that. I've cleaned it up and merged the latest changes from the main branch.

File: blocks/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  box-shadow: inset 0 1px 1px rgba(0 0 0 / 7.5%), 0 0 8px rgba(102 175 233 /60%);
+}
+
+.banner .spotlight-button {
+  background-color: var(--btn-bg-color-3);
+  color: var(--background-color);
+  margin-bottom: 20px;
+}
+
+.banner .spotlight-form-button {
```
Comment (bhellema) [inline blocks/banner/banner.css:63]: I believe the hubblehomes.com button has a border-radius of 0px;

**Pull request closed without merging**